### ISSUE TITLE
git module's clean_bg/dirty_bg don't seem to work properly

### ISSUE THREAD

**Issue by dmolin** [liamg/bearings#5]
Not sure if it's me but I cannot make clean_bg and dirty_bg to work properly in the git module. It seems the color is only applied to the path separator at the end of the current folder (see image, where I set the dirty_bg color to green for the git and the current project has an untracked file)

As an example, this code:
```
 type: git
  clean_bg: "#279a43"
  dirty_bg: "#772222"
  clean_fg: "#7ceb07"
  dirty_fg: "#efe310"
```

Gives me this when there are no changes: 

![2022-03-25_06-56](https://user-images.githubusercontent.com/861513/160070383-1d1a22e1-7160-44df-899c-6f6b3cf179bc.png)

(The blue you see is the background of my terminal window)
clean_fg and dirty_fg instead works perfectly and as intended.

I believe the proble is in modules/git.go at lines 60 and 63, where the function called should be `e.mc.Bg()` instead of `e.mc.Fg()`. I've issued a PR

OS: Manjaro Linux
Shell: bash

**Comment by liamg** [comment]
Thanks @dmolin - I’ll patch this when I’m at my desk in an hour or so 👍

**Comment by liamg** [comment]
Fixed in #6 - thank you!
